pylintrc

```diff
@@ -34,6 +34,9 @@ load-plugins=
 # multiple time (only on the command line, not in the configuration file where
 # it should appear only once).
 disable=
+    missing-docstring,
```

Comment:
This is by far the worst offender. Eventually it will be nice to round out the documentation, but for now, I'd consider this outside the scope. We can add additional docstrings in a future pass, then remove this line.